js/utils/number.js

```diff
@@ -50,3 +51,26 @@ export function upToFixed(number, decimalPlaces) {
 export function randomInt(min = 0, range = 0) {
   return Math.floor((Math.random() * (range + 1)) + min);
 }
+
+/*
+ * Uses local settings' language or provided optional language to return a localized
+ * string representing the provided number.
+ *
+ * ex: number='30000.05', lang='en-us' => 30,000.05
+ * ex: number='30000.05', lang='ru' => 30 000,05
+ * ex: number='30000.05', lang='es' => 30.000,05
+*/
+export function localizeNumber(number, optionalLang) {
+  if (typeof number !== 'number') {
+    throw new Error('Please provide a number.');
+  }
+
+  let lang = optionalLang;
+
+  if (optionalLang === undefined) {
+    lang = app && app.localSettings && app.localSettings.standardizedTranslatedLang() || 'en-US';
```

Comment:
Do we need to check app and app.localSettings here? We should be able to rely on those existing.

This could be done with a default in the parameters, like this:
`export function localizeNumber(number, lang = app.localSettings.standardizedTranslatedLang() || 'en-US')`

That would optimize things a bit.